src/main/java/com/atlassian/db/replica/internal/aurora/AuroraCluster.java

```diff
@@ -0,0 +1,100 @@
+package com.atlassian.db.replica.internal.aurora;
+
+import java.util.Objects;
+
+import static com.atlassian.db.replica.internal.aurora.AuroraCluster.AuroraClusterBuilder.anAuroraCluster;
+
+public final class AuroraCluster {
+    private final String clusterName;
+    private final String clusterPrefix;
+
+    private AuroraCluster(String clusterName, String clusterPrefix) {
+        this.clusterName = clusterName;
+        this.clusterPrefix = clusterPrefix;
+    }
+
+    public static AuroraCluster parse(String cluster) {
+        Objects.requireNonNull(cluster);
+
+        String[] chunks = splitByLastOccurence("-", cluster);
```

Comment:
Whether this string parsing is Aurora specific? or would it also work on Postgres?